.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Build Go package (release)
+        run: ./build
+        if:  ${{ startsWith(github.ref, 'refs/tags/v') }}
+      - name: Build Go package (snapshot)
+        run: ./build --snapshot
```

Comment:
Is the `--snapshot` flag used in the `build` script?

Reply:
It's passed to goreleaser, which takes it as a flag for non-release builds

Reply:
Ah, thanks!